src/server.test.ts

```diff
@@ -19,7 +19,7 @@ jest.mock('./api/contributionsApi', () => {
 });
 
 describe('POST /epic', () => {
-    it('should return an epic', async () => {
+    /*     it('should return an epic', async () => {
```

Comment:
does this need removing or uncommenting?

Reply:
I think my plan was to replace with a module equivalent, so will have a look.